### PR TITLE
Fixed CR example

### PR DESCRIPTION
**Pull request by slintes**

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@slintes I can not see CSV update

**Comment by fromanirh** [comment]
it would be nice to have the csv update or to explain in the commit message why it isn't necessary to do so (always regenerated from CRs?)

**Comment by slintes** [comment]
> @slintes I can not see CSV update

stupid me
updated...

File: deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -29,12 +30,13 @@ metadata:
...
             },
             "realTimeKernel": {
-              "repoURL": "http://download-node-02.eng.bos.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.1.1/compose/RT/x86_64/os"
+              "enabled": true
             }
           }
         }
       ]
     capabilities: Basic Install
+    containerImage: REPLACE_IMAGE
```
Comment (slintes) [inline deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml]: where is this suddenly coming from... :thinking:

Comment (cynepco3hahue) [reply deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml]: I think @davidvossel said he will check from where does it appear.

Comment (slintes) [reply deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml]: Let's blame @fromanirh ;)
https://github.com/openshift-kni/performance-addon-operators/commit/be0dc8dcc1994b08c924e6a881b1a2964c64d451
So all good :)
But we should add a CI check if the CSV is updated when running `make generate-latest-dev-csv`

Comment (fromanirh) [reply deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml]: yep, this is intentional.

**Comment by fromanirh** [comment]
/hold
please remove when ready

File: deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml (hunk 1 above)
Comment (cynepco3hahue) [reply deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml]: yep will add the check

**Pull request merged**